src/args.rs

```diff
@@ -23,7 +23,7 @@ use crate::{
 
 const ALL_NODES_QUERY: &str = "(_) @node";
 
-#[derive(Parser)]
+#[derive(Clone, Parser)]
```

Comment:
I don't know if the need to do `args.clone()` is a "smell" that the crate-level API entry points should be taking an `&Args` instead?